var/spack/repos/builtin/packages/pika/package.py

```diff
@@ -104,6 +104,12 @@ class Pika(CMakePackage, CudaPackage, ROCmPackage):
     patch("transform_mpi_includes.patch", when="@0.3.0 +mpi")
     patch("mimalloc_no_version_requirement.patch", when="@:0.5 malloc=mimalloc")
 
+    # Fix missing template instantiation on macOS
+    patch(
+        "https://github.com/pika-org/pika/commit/dd1dfb85781ec2e76fa37ce7311323e69fbe42a1.patch?full_index=1",
+        when="0.7 platform=darwin",
```

Comment:
Probably pika does not plan to have version 0.7.1, but since it has not been released yet, I think this should apply just to `0.7.0`, otherwise `0.7` will match any other other patch version with minor=7.

Reply:
Yeah, fair point. I'll change that.

Reply:
Pushed.

I was also missing `@` from the version constraint...